data/mods/Magiclysm/worldgen/orc_village.json

```diff
@@ -0,0 +1,65 @@
+    "terrain": { "3": "t_door_makeshift_c", "1": "t_wall_rammed_earth", ".": "t_dirtfloor" }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "object": {
+      "fill_ter": "t_dirt",
+      "om_terrain": [ "orc_village" ],
+      "rows": [
+        "111111111111111111111111111111111111111111111111111111111",
+        "1.......................................................1",
+        "1.......................................................3",
+        "1.......................................................1",
+        "1.......................................................3",
+        "1.......................................................1",
+        "1.......................................................1",
+        "1.......................................................1",
+        "1.......................................................1",
+        "1.......................................................1",
+        "1.......................................................1",
+        "1.......................................................1",
+        "1.......................................................1",
+        "111111111111111111111111111111111111111111111111111111111"
+      ],
+      "palettes": [ "orc_palette" ],
+      "place_monsters": [
+        { "monster": "GROUP_ORC", "x": [ 5, 5 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] },
+        { "monster": "GROUP_ORC", "x": [ 5, 45 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] },
+        { "monster": "GROUP_ORC", "x": [ 5, 45 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] },
+        { "monster": "GROUP_ORC", "x": [ 5, 45 ], "y": [ 5, 46 ], "density": 0.2, "repeat": [ 1, 3 ] }
+      ],
+      {
+    "type": "overmap_special",
+    "id": "orc_village",
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "orc_village" },
+      { "point": [ 0, 0, 1 ], "overmap": "orc_village" },
+      { "point": [ 1, 0, 0 ], "overmap": "orc_village" },
+      { "point": [ 1, 0, 1 ], "overmap": "orc_village" },
+      { "point": [ 2, 0, 0 ], "overmap": "orc_village" },
+      { "point": [ 2, 0, 1 ], "overmap": "orc_village" },
+      { "point": [ 3, 0, 0 ], "overmap": "orc_village" },
+      { "point": [ 3, 0, 1 ], "overmap": "orc_village" },
+      { "point": [ 4, 0, 0 ], "overmap": "orc_village" },
+      { "point": [ 4, 0, 1 ], "overmap": "orc_village" },
+      { "point": [ 0, 1, 0 ], "overmap": "orc_village" },
+      { "point": [ 0, 1, 1 ], "overmap": "orc_village" },
+      { "point": [ 1, 1, 0 ], "overmap": "orc_village" },
+      { "point": [ 1, 1, 1 ], "overmap": "orc_village" },
+      { "point": [ 2, 1, 0 ], "overmap": "orc_village" },
+      { "point": [ 2, 1, 1 ], "overmap": "orc_village" },
+      { "point": [ 3, 1, 0 ], "overmap": "orc_village" },
+    ],
+    "locations": [ "wilderness" ],
+    "city_distance": [ 20, -5 ],
+    "city_sizes": [ 0, 13 ],
+    "occurrences": [ 0, 6 ]
+  }
+    }
+  }
```

Comment:
```suggestion
```
These need to be above the overmap_special - they are closing the mapgen section.